packages/prisma/migrations/20240131021824_revert_20230404202721_add_feature_flag_managed_event_types/migration.sql

```diff
@@ -0,0 +1,3 @@
+-- revert 20230404202721_add_feature_flag_managed_event_types 
+
```

Comment:
Why are we deleting this feature flag?

Reply:
yes, i am ok removing this. this was used to roll it out slowly but now managed event types is a core part of our offering and doenst need a feature flag imo